rl_games/common/player.py

```diff
@@ -15,12 +15,12 @@ def __init__(self, params):
         self.clip_actions = config.get('clip_actions', True)
 
         if self.env_info is None:
-            self.env = self.create_env()
-            self.env_info = env_configurations.get_env_info(self.env)
+            self.env = vecenv.create_vec_env(self.env_name, self.config['num_actors'], **self.env_config)
```

Comment:
Need to think how to make it work with old configs  from openai gym.
Could you test if its ok to open 16 windows with atari for example?
Also it would be nice to be able to override the value from the player config (self.player_config).
In this case we can choose different number of actors if it was set.

Reply:
What do you mean by `old configs from OpenAI Gym`?
If we can't create environments with vec env the API is different between test and training time, and if you want to do wrapping of environments it makes it difficult.

Reply:
Where are we at with this change? @ArthurAllshire did you have a chance to test it with non-IG envs and verify it doesn't break anything?